aiohttp/connector.py

```diff
@@ -386,20 +389,25 @@ def _cleanup_closed(self) -> None:
                 self, '_cleanup_closed',
                 self._cleanup_closed_period, self._loop)
 
-    def close(self) -> Awaitable[None]:
+    async def close(self) -> None:
         """Close all opened transports."""
-        self._close()
-        return _DeprecationWaiter(noop2())
+        waiters = self._close_immediately()
+        if waiters:
+            await asyncio.gather(*waiters,
+                                 loop=self._loop,
+                                 return_exceptions=True)
```

Comment:
possibly re-raise ? I'm unusre

Reply:
Ineed, we lost exceptions here. But maybe we should just log (as warnings) them instead?

Reply:
Heh, very interesting question.
Python and asyncio itself have no MultipleException yet.
Maybe we add it in Python 3.9.
I'm not sure if `warnings.warn()` is a good choice here: it is not a programming error but network issue.
Please use `logging.error()` instead.

Reply:
ok